manifests/noderesources_allocatable/scheduler-config.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kubescheduler.config.k8s.io/v1beta1
```

Comment:
Can we simplify the folder name from `noderesources_allocatable` to `noderesources`?

Reply:
done